### ISSUE TITLE
Initial camera position/focal point not behaving as expected

### ISSUE THREAD

**Issue by tamaynard** [Kitware/react-vtk-js#96]
Attempting to set the initial camera position & focal point for the View component results in unexpected behavior. It appears as if the [initial resetCamera call](https://github.com/Kitware/react-vtk-js/blob/518220649f58374892815f80062e74e83a9fcc37/src/core/View.js#L394) is overriding the provided camera properties. Attempting to set the camera position in user code manually after the component has been initialized results in a blank screen. Is this a bug, and if not is there any way to set set custom initial camera position/focal point on View initialization?

**Comment by floryst** [comment]
Yes, that is unfortunately the case. One solution is to conditionally reset the camera if there isn't any specified camera props.

FYI I'm working on a react 18 upgrade that should address this issue.
